docs/develop/03_workshop/04_attester/02_did.md

```diff
@@ -117,15 +104,17 @@ Now run the code with:
   </TabItem>
 </Tabs>
 
-Once you have run the script, the output should provide you with your `ATTESTER_DID_MNEMONIC` and `ATTESTER_DID_URI`.
-The output should look like the following, but not identical since the DIDs are constructed from your account:
+Once you have run the script, the output should provide you with the `ATTESTER_DID_MNEMONIC` and `ATTESTER_DID_URI`.
+Using the `createFromAccount` method means that the `ATTESTER_DID_MNEMONIC` and `ATTESTER_ACCOUNT_MNEMONIC` are the same.
```

Comment:
So it seems like the script now simply reads the supplied mnemonic back to the user, which I don't think that it makes a lot of sense to do. I'd say we best remove the mnemonic bit from `createFullDid`. And we could replace `ATTESTER_DID_MNEMONIC` with `ATTESTER_ACCOUNT_MNEMONIC` everywhere.

Reply:
Do you think it still makes sense to mention the ATTESTER_DID_MNEMONIC at all? Should we not remove it from the docs and the code when it's identical to the ATTESTER_ACCOUNT_MNEMONIC ?

Reply:
Good call, will remove